projects/gui/src/mainwindow.cpp

```diff
@@ -898,6 +908,19 @@ QString MainWindow::genericTitle(const TabData& gameData) const
 		       .arg(white, black, result.toShortString());
 }
 
+void MainWindow::updateMenus()
+{
+	QPointer<ChessPlayer> white = m_players[Chess::Side::White];
+	QPointer<ChessPlayer> black = m_players[Chess::Side::Black];
+	bool isHumanGame =  (!white.isNull() && white->isHuman())
+			 || (!black.isNull() && black->isHuman());
+	bool gameOn = (!m_game.isNull() && !m_game->isFinished());
+	m_adjudicateBlackWinAct->setEnabled(gameOn);
+	m_adjudicateWhiteWinAct->setEnabled(gameOn);
+	m_adjudicateDrawAct->setEnabled(gameOn);
+	m_resignGameAct->setEnabled(gameOn & isHumanGame);
```

Comment:
I think you meant to use the logical `&&` operator instead of the bitwise `&`?

Reply:
Indeed, this should be the && operator. Thank you! :-)

Reply:
changed, thank you!